smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/i18n/KafkaLogging.java

```diff
@@ -139,8 +139,11 @@
     void configuredPattern(String channel, String pattern);
 
     @LogMessage(level = Logger.Level.WARN)
-    @Message(id = 18231, value = "The amount of received messages without acking is too high for topic partition '%s', amount %d.")
-    void receivedTooManyMessagesWithoutAcking(String topicPartition, long amount);
+    @Message(id = 18231, value = "The amount of received messages without acking is too high for topic partition '%s', "
+            + "amount %d. The last committed offset was %d. It means that the Kafka connector received Kafka Records that "
+            + "have neither be acked nor nacked in a timely fashion. The connector accumulates records in memory, but that "
+            + "buffer reached its maximum capacity. The connector cannot commit as a record processing has not completed.")
```

Comment:
I'm not sure if the remark about the maximum capacity is always true. In my case, only the timeout hit (I have 89 messages waiting in batching). Not sure if you should change the wording.

Reply:
You are right, there is also a deadline.
I fixed it.